Library/Formula/ddar.rb

```diff
@@ -8,11 +8,14 @@ class Ddar < Formula
   head "https://github.com/basak/ddar.git"
 
   depends_on "xmltoman" => :build
-  depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on :python
```

Comment:
One other aspect I'd like to discuss here: `ddar` is not a library for Python developers. It is a command line utility which happens to be implemented in Python. More like `tar` or `gzip`, and less like `pyqt` or `protobuf`'s Python bindings. Its use of python is an implementation detail.

Maybe Homebrew's policy about these things – CLI utilities with Python internals – should be different from libraries with Python bindings or dependencies. Maybe we should be more aggressive about doing what's needed to make it "just work" without the user having to make Python-specific configuration changes to their account (which could have side effects on other programs). Depending on brewed Python, where we can guarantee `/usr/lib/.../site-python` inclusion of its deps, would be one way to do that, though it feels pretty heavyweight. As is, the other option is giving the user a somewhat-obscure warning about Python modules and paths... if they don't already know Python, that seems like a burden.

I don't really know Python and I'm a Homebrew newbie, so I'm definitely open to advice and other views here. But I think `ddar` might be a case where we should try to make it "just work" instead of be efficient.